src/config/index.js

```diff
@@ -143,6 +143,10 @@ const store = new Confidence.Store({
         weather      : {
             openWeatherApiKey : { $env : 'OPEN_WEATHER_API_KEY' },
             LocationIQApiKey  : { $env : 'LOCATION_IQ_API_KEY' }
+        },
+        googleImages    : {
+            apiKey   : { $env : 'GOOGLE_CSE_API_KEY' },
+            engineId : { $env : 'GOOGLE_CSE_ENGINE_ID' }
         }
```

Comment:
Should be under a `tool` attribute

```suggestion
        tool : {
        	googleImages    : {
            	apiKey   : { $env : 'GOOGLE_CSE_API_KEY' },
            	engineId : { $env : 'GOOGLE_CSE_ENGINE_ID' }
     	   }
        }
```